src/main/kotlin/string/calculator/enum/Operator.kt

```diff
@@ -0,0 +1,20 @@
+package string.calculator.enum
+
+enum class Operator {
+    PLUS,
+    MINUS,
+    MULTIPLY,
+    DIVIDE;
+
+    companion object {
+        const val NOT_SUPPORTED_OPERATOR = "지원하지 않는 연산자입니다."
+
+        fun of(operator: String): Operator = when (operator) {
+            "+" -> PLUS
+            "-" -> MINUS
+            "*" -> MULTIPLY
+            "/" -> DIVIDE
```

Comment:
Operator enum class가 `+`, `-`, `*`, `/`와 같은 기호를 프로퍼티로 가진다면
혹시나 새로운 enum 요소(ex. POW)가 생기더라도 정적 팩토리 매서드인 of는 수정을 하지 않을 수 있을 것 같아요. 🤔

Reply:
```suggestion
enum class Operator(val symbol: String) {
    PLUS("+"),
    MINUS("-"),
    MULTIPLY("*"),
    DIVIDE("/");

    companion object {
        private const val NOT_SUPPORTED_OPERATOR = "지원하지 않는 연산자입니다."

        fun of(operator: String): Operator = values().firstOrNull { it.symbol == operator }
            ?: throw IllegalArgumentException(NOT_SUPPORTED_OPERATOR)
    }
}
```
로 변경했습니다. symbol이라는 property를 추가하고 이터레이터로 순회하며 내가 원하는 op가 있나 찾도록 했는데, 경록님이 의도하신 피드백이 맞을까요?

Reply:
지금의 방식 말고도 다른 방식이 있을 수는 있겠지만?  제가 의도했던 방식은 지금과 같은 방식이 맞습니다. 😃